zaza/openstack/charm_tests/ceph/tests.py

```diff
@@ -1038,7 +1038,178 @@ def test_003_object_storage_and_secondary_block(self):
         zaza_model.block_until_unit_wl_status(self.secondary_rgw_unit,
                                               'active')
 
-    def test_004_migration_and_multisite_failover(self):
+    def test_004_multisite_directional_sync_policy(self):
```

Comment:
The new testcase added in [This PR](https://github.com/openstack-charmers/zaza-openstack-tests/pull/1176/) is also numbered `004`. The numbering here is used to run the tests in a particular order and It would be preferable to have non-conflicting test numbers.